AUTHORS.txt

```diff
@@ -4,5 +4,7 @@ Philipp Wolfer <ph.wolfer@googlemail.com>
 Sambhav Kothari <sambhavs.email@gmail.com>
 
 Picard logo by <http://monkeydo.digital>, © 2015 MetaBrainz Foundation
-Interface icons by Carlin Mangar <openshift@gmail.com> and from
-the Tango Desktop Project <http://tango.freedesktop.org/>.
+
+Picard v2.0 interface icons made by Sambhav Kothari <sambhavs.email@gmail.com> and Madebyoliver, Pixel Buddha, Nikita Golubev, Maxim Basinski from www.flaticon.com
```

Comment:
http://support.flaticon.com/hc/en-us/articles/207248209-How-I-must-insert-the-attribution- says 

> For apps:
> Place the attribution on the credits/description page of the application.

which this most likely doesn't fulfill, especially considering that we don't ship AUTHORS.txt.